app/index.js

```diff
@@ -24,34 +28,34 @@ module.exports = yeoman.generators.Base.extend({
     yeoman.generators.Base.apply(this, arguments);
 
     this.argument('name', {
-      desc: 'Name of the application to scaffold.',
+      desc: g.f('Name of the application to scaffold.'),
       required: false,
       type: String
     });
 
     this.option('skip-install', {
-      desc: 'Do not install npm dependencies.',
+      desc: g.f('Do not install npm dependencies.'),
       type: Boolean
     });
 
     this.option('skip-next-steps', {
-      desc: 'Do not print "next steps" info',
+      desc: g.f('Do not print "next steps" info'),
       type: Boolean
     });
 
     this.option('explorer', {
-      desc: 'Add Loopback Explorer to the project (true by default)',
+      desc: g.f('Add Loopback Explorer to the project (true by default)'),
       type: Boolean
     });
   },
 
   greet: function() {
-    this.log(yosay('Let\'s create a LoopBack application!'));
+    g.log(yosay(g.f('Let\'s create a LoopBack application!')));
```

Comment:
This things:
1. `this.log` may not be `console.log`
2. Even if `this.log` is `console.log`,   `g.log(yosay( ...` is not an error and works fine, but won't have nothing to extract, thus, no need to globalize.  Just `this.log(yosay(g.f('Let\'s create a LoopBack application!')));` is the best globalization here.